libusb/os/windows_usbdk.c

```diff
@@ -435,6 +435,14 @@ static void usbdk_close(struct libusb_device_handle *dev_handle)
 	priv->redirector_handle = NULL;
 }
 
+static int usbdk_get_platform_device_id(struct libusb_device *dev, char **data)
+{
+	(void)dev;
```

Comment:
Is this just for silencing warning "parameter not used"? Because I think the standard here is to use "UNUSED(dev)" defined here: https://github.com/libusb/libusb/blob/7313527f23432fdf62ca8caad187341ba23f877b/libusb/libusbi.h#L75

Please consider modifying this.